integrations/terraform/envs/platform/modules.tf

```diff
@@ -0,0 +1,62 @@
+module "account" {
```

Comment:
Do we still need account, networking, infra and stuff? If not, can we remove it?